src/NSwag.AspNetCore/SwaggerUi3Settings.cs

```diff
@@ -39,6 +39,18 @@ public class SwaggerUi3Settings<T> : SwaggerUiSettingsBase<T>
         /// <summary>Specifies the operations sorter in Swagger UI 3.</summary>
         public string OperationsSorter { get; set; } = "none";
 
+        /// <summary>The default expansion depth for models (set to -1 completely hide the models) in Swagger UI 3.</summary>
+        public int DefaultModelsExpandDepth { get; set; } = 1;
+
+        /// <summary>The default expansion depth for the model on the model-example section in Swagger UI 3.</summary>
+        public int DefaultModelExpandDepth { get; set; } = 1;
+
+        /// <summary>Specifies the tag sorter in Swagger UI 3</summary>
+        public string TagSorter { get; set; } = "none";
+
+        /// <summary>Specifies whether the "Try it out" option is disabled in Swagger UI 3.</summary>
+        public bool DisableTryItOut { get; set; } = false;
```

Comment:
I think it would be better to rename this to EnableTryItOut and set to true (avoid negative bools).

Reply:
Can do.